src/tests/ftest/util/scrubber_test_base.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+"""
+(C) Copyright 2021 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import threading
+
+from scrubber_utils import ScrubberUtils
+from ior_test_base import IorTestBase
+
+
+class TestWithScrubber(IorTestBase):
+    # pylint: disable=too-few-public-methods,too-many-ancestors
+    """Test with scrubber enabled.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a Test object."""
+        super().__init__(*args, **kwargs)
+        self.scrubber = None
+
+    def setUp(self):
+        """Set up each test case."""
+        super().setUp()
+        self.scrubber = ScrubberUtils()
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'dmg' in constructor call
(pylint-no-value-for-parameter) No value for argument 'servers' in constructor call